cms/static/sass/views/_outline.scss

```diff
@@ -638,6 +638,12 @@
   .highlights-button {
     cursor: pointer;
     color: theme-color("primary");
+
+    // remove button styling
+    border: none;
+    background: none;
+    padding: 0;
+    font-weight: 600;
```

Comment:
Is there already a style for no buttons?  I vaguely remember there being one - but unsure about Studio.

Reply:
Good call, I found `%ui-btn-non` that does this. It also applies button styling on hover/focus which looks like this:

![screenshot-20171101182525-342x101](https://user-images.githubusercontent.com/1505923/32300961-377ecd92-bf32-11e7-859f-5ee65d11ba24.png)

Reply:
I ended up reverting that change. We didn't like how the gray background on hover effect looked.